### PR TITLE
add information when no Viewport Input node

### PR DESCRIPTION
**Pull request by samytichadou**
tell the user there is no Viewport Input node instead of a blank panel

### CODE REVIEW

**Comment by samytichadou** [comment]
Before : 
image

After : 
image

File: animation_nodes/ui/data_input_panel.py [hunk 1]
```diff
@@ -13,18 +13,21 @@ class ViewportInputPanel(bpy.types.Panel):
     def draw(self, context):
         layout = self.layout
         nodes = getNodesByType("an_ViewportInputNode")
-        for node in nodes:
-            box = layout.box()
+        if len(nodes) == 0:
```
Comment (OmarEmaraDev) [inline animation_nodes/ui/data_input_panel.py]: Can you use a guard condition instead?

Comment (samytichadou) [reply animation_nodes/ui/data_input_panel.py]: you mean like 
```
if len(nodes) == 0:
    layout.label(text="No Viewport Input node", icon="INFO")
    return
for node in nodes:
    box = layout.box()
```

Comment (samytichadou) [reply animation_nodes/ui/data_input_panel.py]: done ! not a really good coder here, i'm not used to this syntax, is it just a readability issue, or does it improve code performance ?

Comment (OmarEmaraDev) [reply animation_nodes/ui/data_input_panel.py]: It probably doesn't affect performance, but it improves readability.

File: animation_nodes/ui/data_input_panel.py [hunk 2]
```diff
@@ -13,13 +13,17 @@ class ViewportInputPanel(bpy.types.Panel):
     def draw(self, context):
         layout = self.layout
         nodes = getNodesByType("an_ViewportInputNode")
+        if len(nodes) == 0:
+            layout.label(text="No Viewport Input node", icon="INFO")
```
Comment (OmarEmaraDev) [inline animation_nodes/ui/data_input_panel.py]: Can you add a period at the end?

**Pull request merged**